suites/pacific/rgw/tier-1-upgrade_4x_to_5x_latest.yaml

```diff
@@ -0,0 +1,412 @@
+#############################################################################
+# - Automation support for upgrade from RHCS4 to RHCS5 cluster with RGW tests
+#
+# Test steps:
+# ---------------------------------------------------------------------
+# - Deploy bare-metal Nautilus Ceph cluster using CDN RPMS with lvm osd scenario
+# - Run RGW tests
+# - Convert the cluster to containerized using ceph-ansible
+# - Upgrade to Pacific using ceph-ansible
+# - Run RGW tests after upgrade
+#
+#############################################################################
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
```

Comment:
Please switch to using `true | false`... i.e. bool is all small cases.

Reply:
Done